pkgs/applications/editors/vim/plugins/nvim-treesitter/generated.nix

```diff
@@ -2555,12 +2555,12 @@
   };
   sql = buildGrammar {
     language = "sql";
-    version = "0.0.0+rev=a966446";
+    version = "0.0.0+rev=b817500";
     src = fetchFromGitHub {
       owner = "derekstride";
       repo = "tree-sitter-sql";
-      rev = "a9664463580473e92d8f5e29fa06fb1be88752af";
-      hash = "sha256-0SY6dOofB+zv4xa7oXabEoUZd5NUV1NHhB+Jx6m137I=";
+      rev = "b8175006d9c8120d41cf40a4ef3711bbbbc08973";
+      hash = "sha256-idQB8Wqw7lvU192y7+UgFvcwlmY71/mu9jJ4hRc4ud4=";
```

Comment:
@GaetanLepage I get hash mismatch here 
```
       specified: sha256-idQB8Wqw7lvU192y7+UgFvcwlmY71/mu9jJ4hRc4ud4=
            got:    sha256-ZURUEd7TokBIVVejRoBiXMQ1XwUEgMnzOhQiV+Tdpk0=
error: 1 dependencies of derivation '/nix/store/gkj82kn7lv4npd2y312dxf47q6l07mb7-sql-grammar-0.0.0+rev=b817500.drv' failed to build
```

Reply:
I tried to reproduce the failure but wasn't able to...
```
trying https://github.com/derekstride/tree-sitter-sql/archive/b8175006d9c8120d41cf40a4ef3711bbbbc08973.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  806k  100  806k    0     0   647k      0  0:00:01  0:00:01 --:--:-- 1064k
evaling implicit 'postFetch' string hook
unpacking source archive /build/b8175006d9c8120d41cf40a4ef3711bbbbc08973.tar.gz
calling 'unpackCmd' function hook '_defaultUnpack' /build/b8175006d9c8120d41cf40a4ef3711bbbbc08973.tar.gz
error: hash mismatch in fixed-output derivation '/nix/store/5fksaff7shpb8qwxa01mf43ybivvdbww-source.drv':
         specified: sha256-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=
            got:    sha256-idQB8Wqw7lvU192y7+UgFvcwlmY71/mu9jJ4hRc4ud4=
```

Reply:
@vtuan10, have you been able to reproduce this ?

Reply:
I can reproduce this with
```
❯ nix-shell -p nix-prefetch-git jq --run "nix hash to-sri sha256:\$(nix-prefetch-git --url https://github.com/derekstride/tree-sitter-sql --quiet --rev b8175006d9c8120d41cf40a4ef3711bbbbc08973 | jq -r '.sha256')"
sha256-ZURUEd7TokBIVVejRoBiXMQ1XwUEgMnzOhQiV+Tdpk0=
```

Reply:
> @vtuan10, have you been able to reproduce this ?

Yes, I encountered this error as well when I updated my NixOS config. It also seems like there exists a PR already which fixes this issue: https://github.com/NixOS/nixpkgs/pull/332824